lib/vintage_net_mobile/modem/huawei_E3372/modemswitch.ex

```diff
@@ -0,0 +1,30 @@
+
+defmodule VintageNetMobile.Modem.HuaweiE3372.Modemeswitch do
+
+  @behaviour VintageNet.PowerManager
+
+  @impl VintageNet.PowerManager
+  def init(_args) do
+    {:ok, {}}
+  end
+
+  @impl VintageNet.PowerManager
+  def power_on(state) do
+    # Do whatever is necessary to turn the network interface on
+    Toolshed.cmd("usb_modeswitch -v 12d1 -p 14fe -X")
```

Comment:
Maybe the id should be configurable. The current values can still be default values

Reply:
Please switch to `System.cmd/3` so that we don't need a dependency on `:toolshed`, but also so that the result can be checked. I'm not sure what to do on an error, but perhaps we just log it for now.